src/main/flight/pid.c

```diff
@@ -371,6 +402,16 @@ void pidController(const pidProfile_t *pidProfile, const rollAndPitchTrims_t *an
             float DTerm = Kd[axis] * delta * tpaFactor;
             DEBUG_SET(DEBUG_DTERM_FILTER, axis, DTerm);
 
+            // if crashDtermThreshold is non-zero then crash recovery is on, so check for a crash
+            if (crashDtermThreshold > 0) {
+                // temporarily allow the beeper to set crash detection, for testing purposes
+                if (IS_RC_MODE_ACTIVE(BOXBEEPERON)
+                        || (motorMixRange >= 1.0 && ABS(DTerm) > crashDtermThreshold && ABS(errorRate) > crashGyroThreshold)) {
+                    inCrashRecoveryMode = true;
```

Comment:
@martinbudden: In order to facilitate testing, how about making it emit a beep whenever it goes into crash recovery mode? This makes fine tuning the activation conditions a bit easier.